### PR TITLE
[zh-cn] Sync configure-liveness-readiness-startup-probes.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md [hunk 1]
```diff
@@ -172,8 +172,9 @@ kubectl get pod liveness-exec
 
 <!--
 The output shows that `RESTARTS` has been incremented:
+The output shows that `RESTARTS` has been incremented. Note that the `RESTARTS` counter increments as soon as a failed container comes back to the running state:
 -->
-输出结果显示 `RESTARTS` 的值增加了 1。
+输出结果显示 `RESTARTS` 的值已经增加。请注意，一旦失败的容器变回运行状态，`RESTARTS` 计数器就会增加 1：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md:177]:
```suggestion
输出结果显示 `RESTARTS` 的值增加了 1。请注意，一旦失败的容器恢复为运行状态，`RESTARTS` 计数器就会增加 1：
```

**Review by windsonsea — COMMENTED** [review]
Could you give some desc in each PR rather than a very long title...

**Comment by Sea-n** [comment]
Hey, this PR title have 109 chars and meaningless, it would be annoying to all contributors.

I think at least we can use only filename (rather than full file path) if we didn't have better one.
/retitle [zh-cn] Sync configure-liveness-readiness-startup-probes.md

**Pull request merged**